assets/helpers/subscriptions.js

```diff
@@ -30,7 +30,9 @@ export type ComponentAbTest = {
 export type BillingPeriod = 'sixweek' | 'quarter' | 'year' | 'month';
 export type WeeklyBillingPeriod = 'sixweek' | 'quarter' | 'year';
 
-export type PaperBillingPlan = 'everyday' | 'sixday' | 'weekend' | 'sunday';
+export type PaperBillingPlan =
+  'collectionEveryday' | 'collectionSixday' | 'collectionWeekend' | 'collectionSunday' |
+  'deliveryEveryday' | 'deliverySixday' | 'deliveryWeekend' | 'deliverySunday';
```

Comment:
this is replicated from #1233 and will probs result in a merge error, #1233 is the source of truth for all conflicts

Reply:
Do you think 
```
export type DeliveryBillingPlan = Everyday' | 'Sixday' | 'Weekend' | 'Sunday';
export type CollectionBillingPlan = Everyday' | 'Sixday' | 'Weekend' | 'Sunday';
```
might be better?